examples/javademo/src/main/java/ConsumeAsset.java

```diff
@@ -0,0 +1,59 @@
+import java.util.UUID;
+
+import forge_abi.Rpc;
+import io.arcblock.forge.ForgeSDK;
+import io.arcblock.forge.Result;
+import io.arcblock.forge.did.WalletInfo;
+import io.arcblock.forge.extension.BigIntegerExt;
+
+/**
+ * █████╗ ██████╗  ██████╗██████╗ ██╗      ██████╗  ██████╗██╗  ██╗
+ * ██╔══██╗██╔══██╗██╔════╝██╔══██╗██║     ██╔═══██╗██╔════╝██║ ██╔╝
+ * ███████║██████╔╝██║     ██████╔╝██║     ██║   ██║██║     █████╔╝
+ * ██╔══██║██╔══██╗██║     ██╔══██╗██║     ██║   ██║██║     ██╔═██╗
+ * ██║  ██║██║  ██║╚██████╗██████╔╝███████╗╚██████╔╝╚██████╗██║  ██╗
+ * ╚═╝  ╚═╝╚═╝  ╚═╝ ╚═════╝╚═════╝ ╚══════╝ ╚═════╝  ╚═════╝╚═╝  ╚═╝
+ * Author       : shan@arcblock.io
+ * Time         : 2019-11-29
+ * Edited By    :
+ * Edited Time  :
+ * Description  :
+ **/
+class ConsumeAsset extends BaseConfig {
+
+  public static void main(String[] args){
+    ForgeSDK forge = ForgeSDK.Companion.connect("localhost", BaseConfig.serverPort);
```

Comment:
Why do we nest `.connect` under `ForgeSDK`, can the developer just call `ForgeSDk.connect`

Reply:
Java call Kotlin  Object method grammar